src/ctia/http/handler.clj

```diff
@@ -115,36 +115,42 @@
-   {:name "Properties"          :description "Properties operations"}
-   {:name "Casebook"            :description "Casebook operations"}
-   {:name "Sighting"            :description "Sighting operations"}
-   {:name "Identity Assertion"  :description "Identity Assertion operations"}
-   {:name "Bulk"                :description "Bulk operations"}
-   {:name "Metrics"             :description "Performance Statistics"}
-   {:name "Target Record"       :description "Target Record operations"}
-   {:name "Tool"                :description "Tool operations"}
-   {:name "Verdict"             :description "Verdict operations"}
-   {:name "Status"              :description "Status Information"}
-   {:name "Version"             :description "Version Information"}])
+
+(s/defn api-tags
+  [{{:keys [enabled?]} :FeaturesService} :- APIHandlerServices]
+  (->>
+   [[:actor               "Actor"               "Actor operations"]
+    [:asset               "Asset"               "Asset operations"]
+    [:asset-mapping       "Asset Mapping"       "Asset Mapping operations"]
+    [:asset-properties    "Asset Properties"    "Asset Properties operations"]
+    [:attack-pattern      "Attack Pattern"      "Attack Pattern operations"]
+    [:bulk                "Bulk"                "Bulk operations"]
+    [:bundle              "Bundle"              "Bundle operations"]
+    [:campaign            "Campaign"            "Campaign operations"]
+    [:casebook            "Casebook"            "Casebook operations"]
+    [:coa                 "COA"                 "COA operations"]
+    [:datatable           "DataTable"           "DataTable operations"]
+    [:event               "Event"               "Events operations"]
+    [:feed                "Feed"                "Feed operations"]
+    [:feedback            "Feedback"            "Feedback operations"]
+    [:graphql             "GraphQL"             "GraphQL operations"]
+    [:identity-assertion  "Identity Assertion"  "Identity Assertion operations"]
+    [:incident            "Incident"            "Incident operations"]
+    [:indicator           "Indicator"           "Indicator operations"]
+    [:judgement           "Judgement"           "Judgement operations"]
+    [:malware             "Malware"             "Malware operations"]
+    [:metrics             "Metrics"             "Performance Statistics"]
+    [:properties          "Properties"          "Properties operations"]
+    [:relationship        "Relationship"        "Relationship operations"]
+    [:sighting            "Sighting"            "Sighting operations"]
+    [:status              "Status"              "Status Information"]
+    [:target-record       "Target Record"       "Target Record operations"]
+    [:tool                "Tool"                "Tool operations"]
+    [:verdict             "Verdict"             "Verdict operations"]
+    [:version             "Version"             "Version Information"]]
+   (mapv (fn [[k n desc]]
```

Comment:
you're returning a vector with some nil elements here, due to the `when` below

probably undesirable